README.md

```diff
@@ -143,3 +143,18 @@ if (thisRepo.isAwesome()) {
 We got accepted for the GSSoc 2021 and activelty looking for new participants,please feel free to ask questions in the discussion section of this repository and if you want to be our mentor please mention in the discussion.
 
 ### Go to the [Wiki](https://github.com/ayan-biswas0412/gssoc2021-HotelOnTouch/wiki) for more information.
+
+
+## Contributors:
+
+### Credits goes to these people:✨
+
```

Comment:
```suggestion
```
remove unnecessary blank lines

Reply:
Ok !!

Reply:
I have removed unnecessary lines but removing more will make the code unclean and it don't impact the spaces in readme

Reply:
okk